packages/map-template/src/components/Map/MapboxMap/MapboxPlacesHandler.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Handle Mapbox places and add geometry with type and coordinates.
+ *
+ * @param {Object} location
+ */
+function addMapboxPlaceGeometry(location, mapboxAccessToken) {
+    return new Promise((resolve, reject) => {
+        const url = `https://api.mapbox.com/search/searchbox/v1/retrieve/${location.id}?session_token=[GENERATED-UUID]&access_token=${mapboxAccessToken}`;
```

Comment:
I guess we need a proper `session_token` here.

Reply:
Yes, that is in progress